### ISSUE TITLE
Unable to navigate in some text fields

### ISSUE THREAD

**Issue by cartertemm** [mltony/nvda-indent-nav#3]
Some editors, for example [notepad2](http://www.flos-freeware.ch/notepad2.html) have text fields that don't seem to work with the latest version of the add-on.

# Steps to Reproduce

* Grab notepad2. Don't bother with the installer, it isn't necessary. The portable version is enough to demonstrate the issue
* Open notepad2.exe and begin typing text
* Try navigating indent levels or even pressing "alt+NVDA+up arrow" in input help

No idea what changed, but interestingly enough I'm able to use version 1.3 just fine.

# Developer Info

```
Developer info for navigator object:
name: u''
role: ROLE_EDITABLETEXT
states: STATE_MULTILINE, STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_ScintillaIAccessibleScintillaWindowNVDAObject object at 0x0568E8B0>
Python class mro: (<class 'NVDAObjects.Dynamic_ScintillaIAccessibleScintillaWindowNVDAObject'>, <class 'NVDAObjects.IAccessible.scintilla.Scintilla'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.scintilla.Scintilla'>, <class 'NVDAObjects.behaviors.EditableTextWithAutoSelectDetection'>, <class 'NVDAObjects.behaviors.EditableText'>, <class 'editableText.EditableText'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=1169, top=118, width=1687, height=1535)
value: None
appModule: <'appModuleHandler' (appName u'notepad2', process ID 2120) at address 568ef90>
appModule.productName: None
appModule.productVersion: None
TextInfo: <class 'NVDAObjects.window.scintilla.ScintillaTextInfo'>
windowHandle: 591398
windowClassName: u'Scintilla'
windowControlID: 64259
windowStyle: 1410334720
windowThreadID: 9264
windowText: u'\u6854\u7369\u6920\u2073\u2061\u6574\u7473\u0a0d\u4909\u2074\u6168\u2073\u6e69\u6564\u746e\u6465\u7420\u7865\u0d74\u090a\u6109\u646e\u7420\u7865\u2074\u6e69\u6564\u746e\u6465\u6120\u2074\u6e61\u746f\u6568\u2072\u656c\u6576\u0d6c\n'
displayText: u'1This is a test2 It has indented text3 and text indented at another level4'
IAccessibleObject: <POINTER(IAccessible) ptr=0xa6fd388 at 56c85d0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=591398, objectID=-4, childID=0
IAccessible accName: u'\u6854\u7369\u6920\u2073\u2061\u6574\u7473\u0a0d\u4909\u2074\u6168\u2073\u6e69\u6564\u746e\u6465\u7420\u7865\u0d74\u090a\u6109\u646e\u7420\u7865\u2074\u6e69\u6564\u746e\u6465\u6120\u2074\u6e61\u746f\u6568\u2072\u656c\u6576\u0d6c\n'
IAccessible accRole: ROLE_SYSTEM_CLIENT
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None
```

**Comment by mltony** [comment]
I believe I just fixed this - please try v1.10.

**Comment by cartertemm** [comment]
Works like a charm, thanks for the quick fix!
